Lesson1/Dima_Gnatyk/tests.cpp

```diff
@@ -0,0 +1,19 @@
+#include "main.cpp"
```

Comment:
**Never include .cpp files into other headers/sources, even in tests!**
You can move your functions declarations in some header instead.